install.sh

```diff
@@ -321,57 +219,50 @@ __install_mongodb() {
 			;;
 		CentOS)
 			if [ ! -s /etc/yum.repos.d/mongodb-org-3.4.repo ]; then
-				echo -e '[mongodb-org-3.4]\nname=MongoDB Repository\nbaseurl=https://repo.mongodb.org/yum/redhat/$releasever/mongodb-org/3.4/x86_64/\ngpgcheck=1\nenabled=1\ngpgkey=https://www.mongodb.org/static/pgp/server-3.4.asc' | $_ELEVATE tee /etc/yum.repos.d/mongodb-org-3.4.repo > /dev/null
+				echo -e '[mongodb-org-3.4]\nname=MongoDB Repository\nbaseurl=https://repo.mongodb.org/yum/redhat/$releasever/mongodb-org/3.4/x86_64/\ngpgcheck=1\nenabled=1\ngpgkey=https://www.mongodb.org/static/pgp/server-3.4.asc' | tee /etc/yum.repos.d/mongodb-org-3.4.repo > /dev/null
 			fi
 			;;
 	esac
 	__install_packages mongodb-org
 	_MONGO_INSTALLED=true
 }
 
-__build_rita() {
-	curl -L -s -o "$GOPATH/bin/dep" https://github.com/golang/dep/releases/download/v0.3.2/dep-linux-amd64
-	chmod +x "$GOPATH/bin/dep"
-
-	export _RITA_SRC_DIR="$GOPATH/src/github.com/activecm/rita"
-	mkdir -p "$_RITA_SRC_DIR"
-
-	# Get the code from git since the build process is dependent on git
-	git clone http://github.com/activecm/rita "$_RITA_SRC_DIR" > /dev/null 2>&1
-
-	local old_dir="$PWD"
-	cd "$_RITA_SRC_DIR"
-	if [ -n "${_RITA_VERSION+x}" ]; then
-		git checkout $_RITA_VERSION > /dev/null 2>&1
+__install_rita() {
+	# Set _RITA_VERSION if it's not set
+	if [ -z ${_RITA_VERSION+x} ]; then
+		_LATEST_VERSION_COMMAND=\
+'curl -sSL https://api.github.com/repos/activecm/rita/releases/latest | grep tag_name | cut -d"\"" -f4'
+		_RITA_VERSION="$(eval $_LATEST_VERSION_COMMAND)"
 	fi
-	make > /dev/null
-	cd "$old_dir"
-}
 
-__install_rita() {
-	$_ELEVATE mkdir -p "$_CONFIG_PATH"
-	#$_ELEVATE mkdir -p "$_VAR_PATH"
-	$_ELEVATE mkdir -p "$_VAR_PATH/logs"
-	$_ELEVATE chmod 777 "$_VAR_PATH/logs"
+	_RITA_DOWNLOAD_URL="https://github.com/activecm/rita/releases/download/$_RITA_VERSION"
+	_RITA_BINARY_URL="$_RITA_DOWNLOAD_URL/rita"	
+	_RITA_CONFIG_URL="$_RITA_DOWNLOAD_URL/config.yaml"
+	_RITA_LICENSE_URL="$_RITA_DOWNLOAD_URL/LICENSE"
+	
+	curl -sSL $_RITA_BINARY_URL -o $_BIN_PATH/rita
```

Comment:
The other two `curl` commands below quote the params. Might as well here for consistency right?